application-task-ui/src/main/resources/TaskManager/ConfluenceTasksReportMacroBridge.xml

```diff
@@ -0,0 +1,1050 @@
+        <classType>com.xpn.xwiki.objects.classes.StaticListClass</classType>
+      </visibility>
+    </class>
+    <property>
+      <async_cached>0</async_cached>
+    </property>
+    <property>
+      <async_context/>
+    </property>
+    <property>
+      <async_enabled>0</async_enabled>
+    </property>
+    <property>
+      <code>{{velocity}}
+#set ($pages = $xcontext.macro.params.spaces)
+#set ($tags = $wikimacro.parameters.labels)
+#set ($assignees = $xcontext.macro.params.assignees)
+#set ($creator = $xcontext.macro.params.creators)
+#set ($createdAfter = $wikimacro.parameters.createddateFrom)
+#set ($status = $xcontext.macro.params.status)
+#set ($sortBy = $wikimacro.parameters.sortBy)
+#set ($displayColumns = $wikimacro.parameters.columns)
+#set ($rowCount = $wikimacro.parameters.pageSize)
+#set ($paramMap = {})
+#if ("$!pages" != '')
+  #set ($discard = $paramMap.put('pages', "pages='$pages'"))
+#end
+#if ("$!tags" != '')
+  #set ($discard = $paramMap.put('tags', "tags='$tags'"))
+#end
+#if ("$!assignees" != '')
+  #set ($discard = $paramMap.put('assignees', "assignees='${assignees}'"))
+#end
+#if ("$!creator" != '')
+  #set ($discard = $paramMap.put('reporters', "reporters='$creator'"))
+#end
+#if ("$!createdAfter" != '')
+  #set ($createdAfter = $datetool.toDate('dd-MM-yyyy', $createdAfter))
+  #set ($createdAfter = $datetool.format($services.taskmanager.configuration.storageDateFormat, $createdAfter))
+  #set ($discard = $paramMap.put('createdAfter', "createdAfter='$createdAfter'"))
+#end
+#if ("$!status" != '')
+  #if ($status == 'complete')
+    #set ($discard = $paramMap.put('status', "status='Done'"))
+  #elseif ($status == 'incomplete')
+    #set ($discard = $paramMap.put('status', "status='InProgress'"))
+  #end
+#end
+#if ("$!sortBy" != '')
+  #set ($discard = $paramMap.put('sortBy', "sortBy='$sortBy'"))
+#end
+#if ("$!rowCount" != '')
+  #set ($discard = $paramMap.put('limit', "limit='$rowCount'"))
+#end
+#if ("$!displayColumns" != '')
+  ## Confluence: description, duedate, assignee, location, labels,     completedate
+  ## XWiki     : name       , duedate, assignee, owner,    createDate, number
+  #set ($colMap = {
+    "description": "name",
+    "duedate": "duedate",
+    "assignee": "assignee",
+    "location": "owner",
+    "labels": "",
+    "completedate": "completeDate"
+  })
+  #set ($colParam = '')
+  #foreach ($col in $displayColumns.split('\s*,\s*'))
+    #if ("$!colMap.get($col)" != '')
+      #if ("$!colParam" == '')
+        #set ($colParam = $colMap.get($col))
+      #else
+        #set ($colParam = "$colParam,${colMap.get($col)}")
+      #end
+    #end
+  #end
+  #set ($discard = $paramMap.put('columns', "columns='$colParam'"))
+#end
+#set ($taskReportCall = "{{task-report $!paramMap.pages $!paramMap.tags $!paramMap.assignees $!paramMap.reporters $!paramMap.createdAfter $!paramMap.status $!paramMap.sortBy $!paramMap.limit $!paramMap.columns /}}")
```

Comment:
I don't understand why you don't iterate the map, instead of hard-coding each key. See also my previous comment. Iterating the map would allow you to perform the wiki syntax escaping here.